models/tokemak/ethereum/tokemak_ethereum_view_lookup_tokens.sql

```diff
@@ -0,0 +1,233 @@
+{{ config(
+    tags=['dunesql', 'static'],
+    schema='tokemak_ethereum',
+    alias = alias('view_lookup_tokens'),
```

Comment:
Maybe we can shorten the alias?
```suggestion
    alias = alias('lookup_tokens'),
```